community-code/How_to_Monitor_Diagnose_and_Solve_Gradient_Issues_in_Foundation_Models/requirements.txt

```diff
@@ -0,0 +1,18 @@
+# Core ML libraries
+torch==1.10.2+cu115  # CUDA 11.5-compatible version
+transformers==4.38.2
+datasets==3.5.0
+
+# Optional: Logging, tracking, utilities
+neptune==1.15.0
+neptune-scale
+tqdm
+numpy
```

Comment:
Please check if the code works for `numpy>=2.0`. If not, please constrain to `numpy<2.0` here